components/blog-index/server.js

```diff
@@ -46,8 +44,7 @@ function PostPreview(context, blogMeta, lazyLoad = false) {
       ${BlogIndexImageFigure(context, {
         image: blogMeta.image,
         slug: blogMeta.slug,
-        width: 1200,
-        height: 630,
+        height: 200,
```

Comment:
I _think_ @pepelsbey just updated these values.

Reply:
Right, I reverted this. I was seeing some horizontal squashing of images at some widths, but that is gone with the single column display on smaller sizes.